core/src/main/java/io/cucumber/core/plugin/TimelineFormatter.java

```diff
@@ -38,7 +39,6 @@
             "/io/cucumber/core/plugin/timeline/index.html",
             "/io/cucumber/core/plugin/timeline/formatter.js",
             "/io/cucumber/core/plugin/timeline/report.css",
-            "/io/cucumber/core/plugin/timeline/jquery-3.4.1.min.js",
```

Comment:
If I'm not mistaken, `chosen.jquery` is a jquery plugin. That probably have to be updated as well.

Reply:
chosen.jquery is indeed a jquery plugin, but there's no update for it. It's at its latest release (which was a securityfix) released in June 2018. Nowadays the project [marks itself as deprecated](https://github.com/harvesthq/chosen#deprecated) awaiting decisions on their future direction.